### ISSUE TITLE
add react native counter part example like React Router Relay

### ISSUE THREAD

**Issue by chirag04** [learnrelay/learnrelay#43]
Right now there is no `react-router-relay` sort of lib for RN. But a simple guide around this could be useful.

**Comment by schickling** [comment]
Yeah, we should definitely add a React-Native variant as well. Nice idea! 👍
